docs/whats-new.md

```diff
@@ -2,30 +2,55 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
-## deck.gl v9.0.0-beta (In Development)
+## deck.gl v9.0.0
 
 deck.gl 9.0 development is actively ongoing with the goal of releasing a beta version as soon as possible.
 
-Target release date: end of January 2024
+Release date: March 15, 2024
+
+<table style={{border: 0}} align="center">
+  <tbody>
+    <tr>
+      <td>
+        <img style={{maxHeight:200}} src="https://github.com/visgl/deck.gl-data/blob/master/images/whats-new/collision-filter-extension.gif?raw=true" />
+        <p><i>WebGPU ready</i></p>
+      </td>
+      <td>
+        <img style={{maxHeight:200}} src="https://github.com/visgl/deck.gl-data/blob/master/images/whats-new/terrain-extension.gif?raw=true" />
+        <p><i>Category filtering</i></p>
+      </td>
+      <td>
+        <img style={{maxHeight:200}} src="https://github.com/visgl/deck.gl-data/blob/master/images/whats-new/wms-layer.gif?raw=true" />
+        <p><i>FirstPersonController</i></p>
+      </td>
+    </tr>
+  </tbody>
+</table>
 
 ### WebGPU enablement
 
 deck.gl v9 adopts the luma.gl v9 API. This will enable deck.gl to run on WebGPU in future releases, however it does cause some unavoidable breaking changes.
 
+### Category filtering in DataFilterExtension
+
+The [DataFilterExtension](./api-reference/extensions/data-filter-extension) has been enhanced to support filtering based on categories. A new accessor, [getFilterCategory](./api-reference/extensions/data-filter-extension), is added which allows filtering using the new [filterCategories](./api-reference/extensions/data-filter-extension#filtercategories).
+
+### FirstPersonController
+
+The behavior of the [FirstPersonController](./api-reference/core/first-person-controller) has been modified to more accurately reflect the motion of a human, making it more intuitive to navigate a 3D space. In addition, panning is supported.
+
 ### Breaking Changes
```

Comment:
This should be in Upgrade Guide instead

Reply:
This also feels like feature highlight that could have a GIF showing a couple new keyboard controls (+ to go vertical, arrows to walk around). I think OSX screen capture records key strokes.